### PR TITLE
Known vendors

### LINKED ISSUE #59: Add known Signalen vendors to contact page
We should make a list of vendors who are either involved in:

- Hosting Signalen
- Doing implementation support for Signalen
- Actively develop on Signalen

Rational is for new Signalen users to see there are vendors with experience about Signalen.

### PR DESCRIPTION
**Pull request by felixfaassen**
Add known vendors who have experience with deliverying services for Signalen.
Fixes #59

### CODE REVIEW

File: src/pages/contact.en.md [hunk 1]
```diff
@@ -32,6 +32,49 @@ René van Sprang<br />
...
+### Vendors
+
+The following vendors have experience in delivering support, implementation and hosting services for Signalen:
+
+**[MACOIN](https://www.macoin.nl/)**<br />
+Implementation/Consultancy<br />
+
+Schellingwoudegracht 12<br />
+3826 CM Amersfoort<br />
+Telephone: +31(0)6 11047859<br />
```
Comment (Ainali) [inline src/pages/contact.en.md:44]:
```suggestion
Telephone: +31 (0)6 110 478 59<br />
```

File: src/pages/contact.md [hunk 2]
```diff
@@ -32,6 +32,49 @@ René van Sprang<br />
...
+### Leveranciers
+
+De volgende Leveranciers hebben ervaring bij het ondersteunen, implementeren en hosten van Signalen:
+
+**[MACOIN](https://www.macoin.nl/)**<br />
+Implementation/Consultancy<br />
+
+Schellingwoudegracht 12<br />
+3826 CM Amersfoort<br />
+Telefoon: +31(0)6 11047859<br />
```
Comment (Ainali) [inline src/pages/contact.md:44]:
```suggestion
Telefoon: +31( 0)6 110 478 59<br />
```

File: src/pages/contact.en.md [hunk 3]
```diff
@@ -32,6 +32,49 @@ René van Sprang<br />
...
+Hosting/Support<br />
+
+Telephone: +31 (0)20 305 97 50<br />
+Email: [info@true.nl](mailto:info@true.nl)
+
+**[Maykin Media](https://www.maykinmedia.nl/)**<br />
+Development/Consultancy/Implementation/Hosting/Support<br />
+
+Keizersgracht 117<br />
+1015CJ Amsterdam<br />
```
Comment (Ainali) [inline src/pages/contact.en.md:65]:
```suggestion
1015 CJ Amsterdam<br />
```

File: src/pages/contact.md [hunk 4]
```diff
@@ -32,6 +32,49 @@ René van Sprang<br />
...
+Hosting/Support<br />
+
+Telefoon: +31 (0)20 305 97 50<br />
+Email: [info@true.nl](mailto:info@true.nl)
+
+**[Maykin Media](https://www.maykinmedia.nl/)**<br />
+Development/Consultancy/Implementation/Hosting/Support<br />
+
+Keizersgracht 117<br />
+1015CJ Amsterdam<br />
```
Comment (Ainali) [inline src/pages/contact.md:65]:
```suggestion
1015 CJ Amsterdam<br />
```

**Review by Ainali — APPROVED** [review]
Some tiny suggestions of spaces for consistency, but not a blocker, good to go.

**Comment by felixfaassen** [comment]
> Some tiny suggestions of spaces for consistency, but not a blocker, good to go.

Thanks @Ainali for your review and suggestions

**Review by Ainali — APPROVED** [review]

**Pull request merged**